packages/core/src/ref-utils.ts

```diff
@@ -43,7 +43,8 @@ export function escapePointer<T extends string | number>(fragment: T): T {
 }
 
 export function parseRef(ref: string): { uri: string | null; pointer: string[] } {
-  const [uri, pointer] = ref.split('#/');
+  const [uri, pointer_] = ref.split('#');
```

Comment:
I don't like the naming though. @malis42 could you suggest a better solution?

Reply:
I think we should do a small refactor in this area and reuse already existing function which is defined below the `parseRef` function, like this:

```ts
export function parseRef(ref: string): { uri: string | null; pointer: string[] } {
  const [uri, pointer = ''] = ref.split('#');

  return {
    uri: uri || null,
    pointer: parsePointer(pointer),
  };
}

export function parsePointer(pointer: string) {
  return pointer.split('/').map(unescapePointer).filter(isTruthy);
}
```